package.json

```diff
@@ -77,7 +78,7 @@
     "react-dom": "16.8.6",
     "react-i18next": "11.7.3",
     "react-redux": "6.0.0",
-    "react-router-dom": "4.3.1",
+    "react-router-dom": "5.2.0",
```

Comment:
I suspect React router is updated for hooks. We've had navigation issues with the Insights Chrome when attempting to update this in the past. Just want to double check that all navigation is working as expected?

Reply:
Yeah, I did remember we had that issue. Anyway, that's not a problem. I can always use "withRouter".